src/terminal/adapter/DispatchTypes.hpp

```diff
@@ -27,6 +27,10 @@ namespace Microsoft::Console::VirtualTerminal::DispatchTypes
         Negative = 7,
         Invisible = 8,
         CrossedOut = 9,
+        // 10 - 19 are actually for selecting fonts, but rarely implemented. 10 and 11
+        // have been repurposed by xterm for XTPUSHGR.
+        SaveForegroundColor = 10,
+        SaveBackgroundColor = 11,

```

Comment:
These really don't belong here, especially since they aren't even being used yet as far I can tell. And if we do want to support the parameterized version of these commands, they should really have their own enum. Some of the values may happen to align with the SGR parameters, but they are by no means identical. And SGR 10 and 11 have defined behaviour that we may well want to support one day.

Reply:
Done.